### PR TITLE
Soft delete cafs

### PR DESCRIPTION
**Pull request by radinreth**

### CODE REVIEW

File: app/models/caf.rb [hunk 1]
```diff
@@ -33,6 +35,8 @@ class Caf < ApplicationRecord
 
   scope :actives, -> { where(actived: true) }
```
Comment (kakada) [inline app/models/caf.rb:36]: @radinreth pleases check whether CAF has been deleted but seems status still remain "active" -> so active will remain include soft-deleted CAF, please confirm

Comment (radinreth) [reply app/models/caf.rb:36]: Once CAF is soft-deleted, the status is meaningless, 
but do you want to update the status to inactive after soft delete right?

Comment (kakada) [reply app/models/caf.rb:36]: @radinreth please check in the scope :actives, -> { where(actived: true) } line above, for the user perspective, when CAFs is being soft-deleted -> Caf.actives should not be included those are soft-deleted, does it make sense?

Comment (radinreth) [reply app/models/caf.rb:36]: right bong, but `act_as_paranoid` does this for us, we don't have to care about it

```sql
Caf.actives.to_sql
SELECT \"cafs\".* FROM \"cafs\" WHERE \"cafs\".\"deleted_at\" IS NULL AND \"cafs\".\"actived\" = TRUE
```

**Review by kakada — APPROVED** [review]

**Pull request merged**